database/gui_repo_worker.py

```
@@ -371,10 +371,16 @@ def get_complete_metadata(self, primary_id: str) -> MetadataInfo:
-            (row[1], row[2], row[3], row[4]) for row in self.cursor.fetchall()
+        reviews: list[ReviewData] = [
+            ReviewData(iteration=row[1], review=row[2], date=row[3])
+            for row in self.cursor.fetchall()
         ]
-        # iteration, rating, review_text, date
+
+        self.cursor.execute(
+            "SELECT rating FROM ratings WHERE comic_id = ?", (primary_id,)
+        )
+        row = self.cursor.fetchone()
+        rating = 0 if not row else row
```

Ruff (0.15.20)

[warning] Use `row if row else 0` instead of `0 if not row else row`

Replace with `row if row else 0`

(SIM212)